testsuite/layers/pom.xml

```diff
@@ -2529,39 +318,2473 @@
                                     </configurations>
                                 </configuration>
                             </execution>
-                            <!-- Core Layers - These can't be tested in Core as they require Undertow -->
-                            <execution>
-                                <id>remoting-provisioning</id>
-                                <goals>
-                                    <goal>provision</goal>
-                                </goals>
-                                <phase>${provisioning.phase}</phase>
-                                <configuration>
-                                    <install-dir>${layers.install.dir}/remoting</install-dir>
-                                    <feature-packs>
-                                        <feature-pack>
-                                            <groupId>${ee.feature.pack.groupId}</groupId>
-                                            <artifactId>${ee.feature.pack.artifactId}</artifactId>
-                                            <version>${ee.feature.pack.version}</version>
-                                            <inherit-configs>false</inherit-configs>
-                                            <inherit-packages>false</inherit-packages>
-                                        </feature-pack>
-                                    </feature-packs>
-                                    <configurations>
-                                        <config>
-                                            <model>standalone</model>
-                                            <name>standalone.xml</name>
-                                            <layers>
-                                                <layer>remoting</layer>
-                                                <layer>undertow</layer>
-                                            </layers>
-                                        </config>
-                                    </configurations>
-                                </configuration>
-                            </execution>
                         </executions>
                     </plugin>
+                </plugins>
+            </build>
+        </profile>
+    </profiles>
+
+            <build>
+                <pluginManagement>
```

Comment:
@jasondlee 
This should be removed, otherwise the plugins are not called. This explains the NPE in the test execution.

In general it seems that, instead of adding the all layers execution in the ee-9 profile, you should add them to the default build and use the provisioning.phase.preview.only property for the phase (this has been put in place but not yet actually used.).

Reply:
@jasondlee , feel free to use the approach you prefer. That was just an possible way (finding that the provisioning.phase.preview.only was defined). Nevertheless I think that the NPE are caused by the pluginManagement. No more execution of the galleon plugins occur.